### ISSUE TITLE
retrain deblending model

### ISSUE THREAD

**Issue by ismael-mendoza** [ismael-mendoza/bliss#100]
- check trained models after 10k epochs are completed. (full sampling figure rerun)
- need to make sure the loss curve is stable
- check `log_every_n_steps` and ensure we only log in the training curve per epoch
- how to make sure validation curve is less spiky?
- increase training data more? 
-
